jsx/PaginationLinks.js

```diff
@@ -69,25 +70,34 @@ let PaginationLinks = React.createClass({
         classList = 'active';
       }
       pageLinks.push(
-        <li key={'table_page_' + i} onClick={this.changePage(i)} className={classList}>
+        <li key={'table_page_' + i.toString()} onClick={this.changePage(i)} className={classList}>
           <a href="#">{i}</a>
         </li>
       );
     }
     if (lastShownPage !== lastPage) {
       pageLinks.push(
-        <li key={'table_page_more'} onClick={this.changePage(lastPage)}>
-          <a href="#">&raquo;</a>
+        <li key={'table_page_more_' + lastShownPage.toString()} onClick={this.changePage(lastPage)}>
+          <a href='#'>&raquo;</a>
         </li>
       );
     }
+
     return (
-            <ul className="pagination pagination-table">
-                {pageLinks}
-            </ul>
-        );
-  },
-});
+      <ul className='pagination pagination-table'>
+          {pageLinks}
+      </ul>
+    );
+  }
+}
+PaginationLinks.propTypes = {
+  onChangePage: PropTypes.func,
+  Total: PropTypes.number.isRequired,
```

Comment:
trailing comma here can be removed

Reply:
@zaliqarosli trailing comma is actually a requirement now with eslint changes.